src/components/Meme/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { PureComponent } from "react";
+
+class Meme extends PureComponent {
+
+  render() {
+    const { memeClass, link } = this.props;
+
+    var extension;
+    var isVideo = false;
+    if(link) {
+      const extensionStartIndex = link.lastIndexOf('.');
+      extension = link.substring(extensionStartIndex + 1, link.length);
+      extension = extension.toLowerCase();
+      if (extension === 'mp4' || extension === 'mp3' || extension === 'ogg' || extension === 'webm') {
+        isVideo = true;
+        extension = 'video/' + extension;
+      }
+
+    }
```

Comment:
this is modularized now but can you move this to utils?
just to make it better?